ddtrace/contrib/trace_utils.py

```diff
@@ -106,12 +107,46 @@ def ext_service(pin, config, default=None):
     return default
 
 
-def set_http_meta(span, config, method=None, url=None, status_code=None):
+def get_error_codes():
+    error_codes = []
+    try:
+        error_str = config.http_server.error_statuses
+    except AttributeError:
+        error_str = None
+    if error_str is None:
+        return [[500, 599]]
+    error_ranges = error_str.split(",")
+    for error_range in error_ranges:
+        values = error_range.split("-")
+        min_code = int(values[0])
+        if len(values) == 2:
+            max_code = int(values[1])
+        else:
+            max_code = min_code
+        if min_code > max_code:
+            tmp = min_code
+            min_code = max_code
+            max_code = tmp
+        error_codes.append([min_code, max_code])
+    return error_codes
+
+
+def set_http_meta(span, integration_config, method=None, url=None, status_code=None, query=None, headers=None):
```

Comment:
`headers` should probably be `request_headers` since we'll want to support both `request_headers` and `response_headers`

Reply:
Should we also do `response_headers` in this PR? It might get a bit too big so it can wait for a "phase 3".

Reply:
I think adding response headers in this PR is good :D